server/package.json

```diff
@@ -22,7 +22,7 @@
     "export": "node scripts/export.js",
     "watch": "NODE_ENV=local nodemon main.js",
     "start": "NODE_ENV=production node main.js",
-    "test": "NODE_ENV=test jest --silent --runInBand --forceExit"
+    "test": "NODE_ENV=test jest --runInBand --forceExit --detectOpenHandles -f psi.service.test.js"
```

Comment:
We've added a `test:debug` in another PR which we can use to test locally, let's keep this as-is to keep the CI checks cleaner